### PR TITLE
GS: update tooltip to remove refernce  to DirectX

### LINKED ISSUE #4705: [BUG]: opengl requires direct3d 10 or 11
### Describe the Bug

in a section labeled "open gl advanced settings" there are two options "geometry shader" and "image load store" whose tooltips say they need directx 10 or 11 respectively

### Reproduction Steps

hover mouse above options

### Expected Behavior

i honestly don't know if the tool tip is wrong or the feature is

### PCSX2 Revision

dev-1676

### Operating System

Windows 10 (32bit)

### If Linux - Specify Distro

_No response_

### CPU

i5-9600

### GPU

gtx 960

### GS Settings

_No response_

### Emulation Settings

_No response_

### GS Window Screenshots

Untitled

### Logs & Dumps

_No response_

### PR DESCRIPTION
**Pull request by Mrlinkwii**
### Description of Changes

update the tooltip not to reference DirectX

### Rationale behind Changes

to make it  the tooltip  more correct ,  fixes #4705 

### Suggested Testing Steps

open/complie    the PR and read the tooltip

### CODE REVIEW

File: pcsx2/GS/Window/GSSetting.cpp [hunk 1]
```diff
@@ -163,11 +163,11 @@ const char* dialog_message(int ID, bool* updateText)
 		case IDC_GEOMETRY_SHADER_OVERRIDE:
 			return cvtString("Allows the GPU instead of just the CPU to transform lines into sprites. This reduces CPU load and bandwidth requirement, but it is heavier on the GPU.\n"
 				"Automatic detection is recommended.\n\n"
-				"Note: This option is only supported by GPUs which support at least Direct3D 10.");
+				);
```
Comment (lightningterror) [inline pcsx2/GS/Window/GSSetting.cpp:166]: You can move this up on the previous row and remove the two \n

File: pcsx2/GS/Window/GSSetting.cpp [hunk 2]
```diff
@@ -163,11 +163,11 @@ const char* dialog_message(int ID, bool* updateText)
...
 		case IDC_IMAGE_LOAD_STORE:
 			return cvtString("Allows advanced atomic operations to speed up DATE Accuracy.\n"
 				"Only disable this if using DATE Accuracy causes (GPU driver) issues.\n\n"
-				"Note: This option is only supported by GPUs which support at least Direct3D 11.");
+				);
```
Comment (lightningterror) [inline pcsx2/GS/Window/GSSetting.cpp:170]: Same.

**Comment by lightningterror** [comment]
Commit can be better named.

File: pcsx2/GS/Window/GSSetting.cpp [hunk 3]
```diff
@@ -163,11 +163,11 @@ const char* dialog_message(int ID, bool* updateText)
 		case IDC_GEOMETRY_SHADER_OVERRIDE:
 			return cvtString("Allows the GPU instead of just the CPU to transform lines into sprites. This reduces CPU load and bandwidth requirement, but it is heavier on the GPU.\n"
 				"Automatic detection is recommended.\n\n"
-				"Note: This option is only supported by GPUs which support at least Direct3D 10.");
+				);
 		case IDC_IMAGE_LOAD_STORE:
 			return cvtString("Allows advanced atomic operations to speed up DATE Accuracy.\n"
 				"Only disable this if using DATE Accuracy causes (GPU driver) issues.\n\n"
```
Comment (lightningterror) [inline pcsx2/GS/Window/GSSetting.cpp:169]: You can change DATE Accuracy to Accurate DATE.

**Comment by tadanokojin** [comment]
geo shader I don't care (we require a feature level 10 gpu at minimum anyway)
image load store you need to specify.

**Review by lightningterror — APPROVED** [review]

**Pull request merged**